generator/src/main/java/com/scottlogic/deg/generator/restrictions/SetRestrictions.java

```diff
@@ -1,43 +1,53 @@
 package com.scottlogic.deg.generator.restrictions;
 
 import com.scottlogic.deg.generator.utils.SetUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 
 import java.util.Collections;
-import java.util.HashSet;
 import java.util.Objects;
 import java.util.Set;
 import java.util.stream.Collectors;
 
 public class SetRestrictions {
-    private static final SetRestrictions neutral = new SetRestrictions();
+    private static final SetRestrictions neutral = new SetRestrictions(null, null);
 
+    @Nullable
```

Comment:
I don't like the nullable, because it's meaningless i stop reading anything that looks like it.

this means i miss the meaningful @ NotNull annotation.